src/install/mongodb/find-data.mdx

```diff
@@ -0,0 +1,23 @@
+---
+componentType: default
+headingText: Find and use data
+---
+
+import infrastructureInstallMysqlDashboard from 'images/infrastructure_screenshot-crop_install-mysql-dashboard.webp'
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > Add data** and type **MongoDB**.
+2. Select **Dashboards**, and click **MongoDB overview dashboard**.
+<img
+  title="Install the MongoDB dashboards"
+  alt="Installing the MongoDB dashboards"
+  src={infrastructureInstallMysqlDashboard}
+/>
+
+3. In the popup window, select your account
+4. Click **View dashboard**, and see your MongoDB data in New Relic.
+
+
+You can find a full list of metrics and their attributes at the bottom of this doc. 
+
+For more on how to find and use your data, see how to [understand integration data](/docs/infrastructure/integrations/find-use-infrastructure-integration-data).
+
```

Comment:
```suggestion
For more on how to find and use your data, see how to [understand integration data](/docs/infrastructure/integrations/find-use-infrastructure-integration-data). You can find a full list of metrics and their attributes at the bottom of this doc. 

```

Reply:
Consider switching these to improve the flow from the last sentence (which is about data in the UI)